packages/iris-grid/src/IrisGridPartitionSelector.tsx

```diff
@@ -327,7 +321,9 @@ class IrisGridPartitionSelector extends Component<
     ));
     return (
       <div className="iris-grid-partition-selector">
-        <div className="table-name">Partitioned Table</div>
+        <div className="table-name">
+          {model.isPartitionRequired ? 'Partitioned Table' : ''}
+        </div>
```

Comment:
This leaves a weird divider on the left side when it's not a partitioned table. Why the empty string?

It looks like the epic ticket says to just remove this label